hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/services/AlertsService.java

```diff
@@ -30,18 +30,25 @@
 public interface AlertsService {
 
     void sendData(Data data);
+
     void sendData(Collection<Data> data);
 
     Collection<Alert> checkAlerts();
```

Comment:
I think is also interesting to start introducing more rich methods for getting the alerts.
From the UI we can have the use case to get the alerts for an specific trigger, and also we should combine the tagging/grouping mechanism of the definitions into the alerts.
Also, filtering by time interval will be something to add.
An, in addition of all below, we also should add getCount() methods to be able to deal with pagination as well.